examples/cluster/plot_digits_linkage.py

```diff
@@ -37,7 +30,8 @@
 
 from sklearn import manifold, datasets
 
-X, y = datasets.load_digits(return_X_y=True)
+digits = datasets.load_digits()
+X, y = digits.data, digits.target
```

Comment:
please revert these, unless the original is much slower.

Reply:
Used the digit variable in `for digit in digits.target_names:` for iterating over targets_names. Previously I was iterating over `y` but it was proposed to change it to current format.

Reply:
I think this is a good intend, but in that case this changes the PR goal which does not only accelerate this example but generally improves it.

I see two way this can be resolved: either you change this PR title or you submit another PR for those presentation changes.

Reply:
My bad, didn't notice that, it's good as is